articles/extensions/custom-social-extensions.md

```diff
@@ -24,7 +25,7 @@ You will be asked to authorize the Custom Social Connections app. After you do s
 

```

Comment:
This image needs to be updated, we have more social providers:

![image](https://cloud.githubusercontent.com/assets/11715799/22368340/aef51324-e48e-11e6-8449-9107f36b6ed5.png)